assets/scss/base/_reused.scss

```diff
@@ -134,3 +138,23 @@
 [data-reveal="right"].revealed {
   transform: translateX(0);
 }
+
+@media (min-width: 768px) {
```

Comment:
define breakpoints as vars

Reply:
It can't be used var in media queries - tested it.
I don't think that below solution is good.
https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries/68788747#68788747


This comment explain that
https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries/68788747#comment99646992_40722882

Reply:
So shall i rewrite all files to sass to use media queries with var? :(

Reply:
Nah, just add the sass vars for media queries